### DISCUSSION TITLE
always waiting for free solvers

### DISCUSSION THREAD

**Discussion by dreamqin68** [dafny-lang/dafny#3270] [Q&A] [answered]
Hi All,

Merry Chrismas!

I installed dafny 3.10.0 in the extension of vscode, and I found that this sentence is always displayed on the bar below the vscode interface:
`Verified 0/1, waiting for free solvers Running tests...` 

I restarted dafny,  but `waiting for free solvers` was still there. Then I waited an hour and it was still there. I don't know why there is no free solver. 

I guess it may be related to the `Verification Virtual Cores`, the default value is `0`.

Thanks in advance.

**Comment by basbebe** [comment]
Same problem here.
Explicitly setting `Verification Virtual Cores` works as a workaround for me (thanks for that!).

**Comment by keyboardDrummer** [comment, marked as answer]
This is a bug, we're working on resolving it. Relevant issue is here: https://github.com/dafny-lang/ide-vscode/issues/326

**Reply by keyboardDrummer** [reply]
Resolved now. Is it OK if I delete this discussion?

**Reply by dreamqin68** [reply]
Thanks! sure.
